packages/vscode-apollo/package.json

```diff
@@ -20,6 +20,7 @@
     "update-vscode": "node ../../node_modules/vscode/bin/install",
     "create-server-symlink": "npx rimraf node_modules && mkdir node_modules && node create-server-symlink.js",
     "package-extension": "npx rimraf node_modules && yarn install --force --production --ignore-scripts && ../../node_modules/vsce/out/vsce package --yarn --baseContentUrl https://raw.githubusercontent.com/apollographql/apollo-tooling/master/packages/vscode-apollo && git clean -dfqX -- ./node_modules ./yarn.lock && npm run create-server-symlink",
+    "publish-extension": "npx rimraf node_modules && yarn install --force --production --ignore-scripts && ../../node_modules/vsce/out/vsce publish -p $VS_MARKETPLACE_TOKEN --yarn --baseContentUrl https://raw.githubusercontent.com/apollographql/apollo-tooling/master/packages/vscode-apollo",
```

Comment:
publish extension _can_ just publish an already built vsix file, but since we don't know the name of it already (and aren't currently renaming it), I thought it'd probably just be easiest to build it again in this step. I'm open to renaming it something predictable and just publishing the already-built artifact from a previous step though. I'd need to figure out how to access artifacts from previous steps too :)

Reply:
Also, is the cleanup bit that currently lives on `package-extension` important for `publish-extension`? (Assuming it's ever ran locally?)  I guess I'm worried that if `yarn.lock` gets left (which for some inexplicable reason is cleansed, rather than using `yarn --no-lockfile`, 🤷‍♂), something weird might happen for the next step.

e.g., this bit:

```
 && git clean -dfqX -- ./node_modules ./yarn.lock && npm run create-server-symlink
```

Reply:
I extracted out those two parts and added an `_internal:prepare-extension` and `_internal:cleanup-extension` just to be safe :) I can revisit all this when I work on bundling the extension in the mid-to-near future